src/nodelets/CameraController.cpp

```diff
@@ -31,5 +33,8 @@ void CameraController::setupFramePublish() {
 
 void CameraController::captureAndPublish(const ros::TimerEvent &evt)
 {
-    publishImage(ros::Time::now());
+    uint64_t nsecs = fmod(_camera->getActualTimestamp(), 1e9);
```

Comment:
should there be a condition here so we can use the ros time if the sensor time sync (ptp) is not enabled?